src/app/gestion-proveedores/gestion-proveedores.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { ProveedoresService } from '../services/proveedores/proveedores.service';
+import {
+  Proveedor,
+  ProveedoresResponse,
+  ProveedorResponse
+} from '../services/proveedores/proveedores.types'
+
+@Component({
+  selector: 'app-gestion-proveedores',
+  templateUrl: './gestion-proveedores.component.html',
+  styleUrls: ['./gestion-proveedores.component.css']
+})
+export class GestionProveedoresComponent implements OnInit {
+  protected proveedores: Proveedor[] = [];
+
+  constructor(
+    private proveedoresService: ProveedoresService,
+    private router: Router
+  ) {}
+
+  ngOnInit(): void {
+    this.ObtenerProveedores();
+  }
+  /**
+   * Esta función llama al servicio de proveedores, obtiene todos los proveedores de la base de datos
+   * y actualiza la lista de proveedores del componente.
+   */
+   protected ObtenerProveedores(): void {
+    this.proveedoresService.todosProveedores().subscribe({
+      next: (proveedoresResponse: ProveedoresResponse) => {
+        if (proveedoresResponse.exito) {
+          this.proveedores = proveedoresResponse.proveedores.sort(
+            (proveedorA, proveedorB) => {
+              return proveedorA.cedula_juridica_proveedor.localeCompare(
+                proveedorB.cedula_juridica_proveedor
+              );
+            }
+          );
```

Comment:
```suggestion
          this.proveedores = proveedoresResponse.proveedores;
```

Esto es para ordenar por orden alfabético, no aplica para cédulas.